### ISSUE TITLE
Network IX adds & deletes are immediate, rather than taking effect on Save

### ISSUE THREAD

**Issue by ccaputo** [peeringdb/peeringdb#808]
While testing the IX-F Importer I noticed the unexpected behavior that when a network is editing their list of IXes, performing an add, or delete, or IX-F Import Hint Add or Auto-Add, results in an immediate modification of the database, rather than pending until the Save button is clicked.  Ie. one could refresh the page or simply view it elsewhere, while still in Edit mode, and the new or removed IX will be listed.

Further, clicking Cancel while in edit mode, does not revert the add or delete as it should.

This would seem to be a violation of UX expectations.

**Comment by grizz** [comment]
Fast track since it's a clear bug.

**Comment by grizz** [comment]
This is going to require a fair amount of rework, so won't be ready for release. I don't think it's a show stopper, is it?

@ccaputo @peeringdb/pc

**Comment by ccaputo** [comment]
> This is going to require a fair amount of rework, so won't be ready for release. I don't think it's a show stopper, is it?
> 
> @ccaputo @peeringdb/pc

Agreed it is not a showstopper, especially since it has been this way well before the IX-F Importer, IIRC.

**Comment by Yo-Robinson** [comment]
Agree, not a show stopper.

**Comment by shane-kerr** [comment]
+1

I don't mind if this goes out in a subsequent release, since it isn't a new problem exactly.

**Comment by leovegoda** [comment]
@peeringdb/pc This was csheduled but is challenging to do in software. Another approach we could try is to improve documentation. Would that would for you?
